cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -232,7 +232,7 @@ public static Secret createInitialCaKeySecret(String clusterNamespace, String cl
                 .build();
     }
 
-    public static Kafka createKafka(String namespace, String name, int replicas,
+    public static Kafka createKafka(String namespace, String name, Integer replicas,
```

Comment:
Do you really need to change this? Do you expect to pass null somewhere to these methdos?